frontend/src/components/question-creation/question-elements/modals/multiple-choice/AddMultipleChoiceModal.tsx

```diff
@@ -15,29 +15,41 @@ const AddMultipleChoiceModal = (): React.ReactElement => {
   const {
     showAddMultipleChoiceModal,
     setShowAddMultipleChoiceModal,
+    showAddMultiSelectModal,
+    setShowAddMultiSelectModal,
   } = useContext(QuestionEditorContext);
   const closeModal = () => {
     setShowAddMultipleChoiceModal(false);
+    setShowAddMultiSelectModal(false);
     onClose();
   };
```

Comment:
Could you update `AddMultipleChoiceModal` -> `AddMultiOptionModal` or something like that? also all the references to **Multiple Choice** in this file! & same for the `EditMultipleChoiceModal` and `MultipleChoiceModal`. You can also update the folder naming!